vendor/licenses/grammar/vscode-slice.txt

```diff
@@ -31,4 +32,4 @@ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
 SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
 CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
\ No newline at end of file
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

Comment:
What happened here?

Reply:
What do you mean? In my recent commit I added the version.

Reply:
This line appears to have been modified, although GitHub's diff-view isn't showing what changed... Strange.

A trailing newline may have been added or removed when you edited the license, but GitHub normally indicates that with an 🚫 icon.

Reply:
Yeah my local diff shows that I added a trailing newline
```
diff --git a/vendor/licenses/grammar/vscode-slice.txt b/vendor/licenses/grammar/vscode-slice.txt
index b1e84f5c..0103ec63 100644
--- a/vendor/licenses/grammar/vscode-slice.txt
+++ b/vendor/licenses/grammar/vscode-slice.txt
@@ -1,6 +1,7 @@
 ---
 type: grammar
 name: vscode-slice
+version: 6a28c2125bb442fa6a7f31c995ce423b1c9bbfb7
 license: bsd-3-clause
 ---
 BSD 3-Clause License
@@ -31,4 +32,4 @@ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
 SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
 CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
\ No newline at end of file
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```